content/applications/sales/sales/amazon_connector/manage.rst

```diff
@@ -78,19 +78,36 @@ and allows you to manage the stock of your products under the FBA program.
 Issue invoices and register payments
 ====================================
 
-You can issue invoices for Amazon orders in Odoo. Click **Create Invoice** in the sales order to do
-so. You can also do it in batch from the list view of orders. Then, confirm and send the invoices to
-your customers.
+Issue invoices
+--------------
+
+Sending invoices to Amazon customers directly from Odoo is not feasible due to Amazon's policy of
+not sharing customer email addresses. Instead, you can manually upload the
+invoices generated on Odoo to the Amazon backend.
+
+In addition, it is currently required to manually retrieve VAT numbers from the Amazon backend
+before creating an invoice due to an issue with the Selling Partner API.
```

Comment:
This does not sounds very clear for a regular user. I don't think they are aware of what is the "Selling Partner API". 

Maybe something like:
"For your B2B clients, you will have to find their VAT number in Amazon before creating the invoice in Odoo."